### DISCUSSION TITLE
Stacktrace is missing in log

### DISCUSSION THREAD

**Discussion by yosigolan** [RehanSaeed/Serilog.Exceptions#89] [Q&A]
Hey.
i am using this package along with the google cloud stackdriver logging sink.
i know that you wrote in the readme file that serilog adds the stacktrace but on google logs it dont shows.
how do i add back the stacktrace property?
![image](https://user-images.githubusercontent.com/18391088/51264303-e0f2cd00-19be-11e9-9be6-b8671f538e9e.png)

Thanks

**Comment by RehanSaeed** [comment]
See https://github.com/RehanSaeed/Serilog.Exceptions/blob/7c940e7c1f64e97849805c08a69dc8c7ecb7c7b2/Source/Serilog.Exceptions/Core/DestructuringOptionsBuilder.cs#L29-L30

You can pass your own configuration that does not call this method https://github.com/RehanSaeed/Serilog.Exceptions/blob/7c940e7c1f64e97849805c08a69dc8c7ecb7c7b2/Source/Serilog.Exceptions/Core/DestructuringOptionsBuilder.cs#L115

**Comment by yosigolan** [comment]
Thank you for the response.
how do i pass a configuration that overrides the defaults? can you give a quick example?
is it possible to configure it using the appsettings file or that i have to use the code?

**Comment by RehanSaeed** [comment]
https://github.com/RehanSaeed/Serilog.Exceptions#additional-configuration

**Comment by yosigolan** [comment]
Thank you it works.
